.github/workflows/ci.yml

```diff
@@ -213,10 +241,18 @@ jobs:
         uses: actions/upload-artifact@v2
         if: failure()
         with:
-          name: result hive-tests
+          name: result ${{ github.job }}
           path: |
             **/target/surefire-reports
             **/target/checkstyle-*
+      - name: Upload test report
+        uses: actions/upload-artifact@v2
+        if: always()
+        with:
+          name: test report ${{ github.job }} (${{ matrix.config }})
+          path: |
+            **/surefire-reports/TEST-*.xml
+          retention-days: 5
```

Comment:
can this be set with a variable, or an env, to have a central switch for retention?

Reply:
Done!